spec/support/cocina_fixtures.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
```

Comment:
We should chat about whether we use this approach or factories or both

Reply:
Yeah, I understand this is not our normal pattern so worth a discussion.